docs-content/general/changelog/changelog-24.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Changelog 24 (09/11)
+slug: changelog-24
+---
+
+## GitHub-enhanced stack traces
+
+Highlight is all about tight integrations. We’ve expanded our GitHub integration to link stack traces directly to the relevant files in your GitHub repo.
+
+It’s easy to enable and even easier to use. 
+
+It seems like a tiny feature at first, but it’ll spoil you. You’ll question how you ever lived without it.
+
+[GitHub Integration Docs](https://www.highlight.io/docs/general/product-features/error-monitoring/enhancing-errors-with-github)
+
+<EmbeddedVideo 
+  src="https://www.loom.com/embed/65b655146ce8441d9769a8d4dc389dcc?sid=d0d3fa30-071f-418f-b35f-9d6cccdfcad1"
```

Comment:
Do we want to use this video if we are sharing with customers?
https://www.loom.com/share/5b362125ffd94e8cba1f442b5fc56ded

Reply:
Great catch. I've committed the change and I'll deploy with it!

Thanks for the review!